CMakeLists.txt

```diff
@@ -745,3 +773,46 @@ if (EMSCRIPTEN)
     LINK_DEPENDS "${WABT_POST_JS};${EMSCRIPTEN_EXPORTED_JSON}"
   )
 endif ()
+
+# Create find_package configuration files
+if (WABT_INSTALL_RULES)
+  include(CMakePackageConfigHelpers)
+
+  set(WABT_INSTALL_CMAKEDIR "${CMAKE_INSTALL_LIBDIR}/cmake/wabt"
+      CACHE STRING "Path to wabt CMake files")
+
+  install(
+    EXPORT wabt-targets
+    DESTINATION "${WABT_INSTALL_CMAKEDIR}"
+    NAMESPACE wabt::
+    FILE wabt-targets.cmake
+    COMPONENT wabt-development
+  )
+
+  file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/wabt-config.cmake.in" [[
+cmake_minimum_required(VERSION 3.8)
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/wabt-targets.cmake")
```

Comment:
Hhere does wabt-targets.cmake come file?

Reply:
That comes from here: 

https://github.com/WebAssembly/wabt/blob/52ec70d09a6ef86eda4995168a5572092c6ba706/CMakeLists.txt#L784-L790